### PR TITLE
test: page.evaluate

### PR DESCRIPTION
**Pull request by yury-s**

### CODE REVIEW

File: playwright/src/main/java/com/microsoft/playwright/impl/FrameImpl.java [hunk 1]
```diff
@@ -244,6 +244,7 @@ public Object evaluate(String expression, Object arg) {
     params.addProperty("world", "main");
     params.addProperty("isFunction", isFunctionBody(expression));
     params.add("arg", gson().toJsonTree(serializeArgument(arg)));
+    System.out.println(new Gson().toJson(params));
```
Comment (kblok) [inline playwright/src/main/java/com/microsoft/playwright/impl/FrameImpl.java:247]: Do we need this?

File: playwright/src/test/java/com/microsoft/playwright/TestPageEvaluate.java [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+  @Test
+  void shouldReturnUndefinedForObjectsWithSymbols() {
+    assertEquals(singletonList(null), page.evaluate("() => [Symbol('foo4')]"));
+    assertEquals(emptyMap(), page.evaluate("() => {\n" +
+      "  const a = {};\n" +
+      "  a[Symbol('foo4')] = 42;\n" +
+      "  return a;\n" +
+      "}"));
+  assertEquals(mapOf("foo", asList(mapOf("a", null))), page.evaluate("() => {\n" +
```
Comment (kblok) [inline playwright/src/test/java/com/microsoft/playwright/TestPageEvaluate.java:138]: I think we have an indent issue here.

File: playwright/src/main/java/com/microsoft/playwright/impl/FrameImpl.java (hunk 1 above)
Comment (yury-s) [reply playwright/src/main/java/com/microsoft/playwright/impl/FrameImpl.java:247]: Oops, reverted.

File: playwright/src/test/java/com/microsoft/playwright/TestPageEvaluate.java (hunk 2 above)
Comment (yury-s) [reply playwright/src/test/java/com/microsoft/playwright/TestPageEvaluate.java]: Done.

**Pull request merged**